src/docs/JobPlanner/references/JobPlannerServiceReference.adoc

```diff
@@ -47,6 +47,22 @@ When selecting a calendar with associated workflows, it is possible to filter th
 
 image::calendar_wf_association_4.png[align="center"]
 
+This filtering can also be applied in the Calendars section on the left. By applying this filtering, it will filter both calendars and associated workflows.
+
+image::calendar_wf_association_5.png[align="center"]
+
+An advanced filtering is available by clicking on the filter icon.
+
+image::calendar_wf_association_6.png[align="center"]
+
+Advanced filtering will update the associations to match the filters but also the calendars and the statistics on the status of planned jobs at the top of the page.
+
+image::calendar_wf_association_7.png[align="center"]
+
+To clear the filters, you can either click on the Clear filters button next to the filters pills, or use the Clear button the advanced filtering form as you can see in the second screenshot above.
```

Comment:
use the Clear button in the advanced filtering

Reply:
this version is better:

To reset the filters, you have two options: you can click on the "Clear icon" located next to the filter pills, or you can use the "Clear button" within the advanced filtering form, as depicted in the second screenshot above